adserver/analyzer/tasks.py

```diff
@@ -33,6 +33,7 @@ def analyze_url(url, publisher_slug, force=False):
     - Creates or updates an entry in AnalyzedUrls table
     - Discovers keywords and topics for a URL
     """
+    skip_renanalyze_days_threshold = 30
```

Comment:
I know we're using these magic numbers in a few places, we should make it a proper setting. 

It should probably update here also: https://github.com/readthedocs/ethical-ad-server/blob/005b2ddf3cd52c33c57bcba458b3f9cb9ce857ba/adserver/analyzer/tasks.py#L168